docs/proposals/nvidia-gpu-enhancement.md

```diff
@@ -0,0 +1,72 @@
+# Add NVIDIA GPU discovery and assignment capabilities to Kubelet.
+
+## Background:
+
+So far we only support 1 GPU(```/dev/nvidia0```) and the GPU device need to be configured as a Kubelet's parameter manually. But in production environment, there could have multiple GPUs(```/dev/nvidia0, /dev/nvidia1, /dev/nvidia2 ...```) installed on 1 Minion, also 1 micro service could needs more than 1 GPU, and there could have thousands of Minions which means if Kubelet could discover GPU by itself would be great.
```

Comment:
Mention of micro-services is extraneous and I have a few wording nits.

Suggest: "Currently, we only support a single GPU, `dev/nvidia0`, on a node. In addition, using this GPU require manual configuration of the Kubelet.

This document proposes implementing support for auto-discovery of GPUs in an extensible manner, support for identifying multiple GPUs, and support for applications which require multiple GPUs."